templates/index.json

```diff
@@ -21,7 +21,7 @@
           "type": "buttons",
```

Comment:
Let's revert this. I think you may need to enable `one sync` with github

Reply:
Actually how do you do this? I think I've always just connected them with two-way sync and can't find the button to change it.

Reply:
That's where I figured it would be, but I don't have that button for some reason 😕 
<img width="1091" alt="image" src="https://github.com/Shopify/dawn/assets/590055/acfd5688-3632-4bd5-bad7-2c3832d3c4b2">

Reply:
Ah I have seen this before - let me message you